examples/tutorials/basics/plot.py

```diff
@@ -1,6 +1,6 @@
 """
 Plotting data points
---------------------
+====================
```

Comment:
It seems we're inconsistently using `----` or `=====` below the title.

Reply:
In gallery examples we are nearly all most using `----` below the title. But for the tutorials and intros, we are using `====` below the title and `----` below the headings of the subsections (if present). For consistency we maybe / probably should update the examples to have `====`  below the title?

Reply:
Let's focus on the empty line in the PR, and decide to use `---` or `===` later.

Reply:
Yes, it makes sense to handle these two aspects separately, as this title underline thing is unrelated to the `tooltip` issue.